apps/antalmanac/src/components/RightPane/CoursePane/CourseRenderPane.tsx

```diff
@@ -50,7 +50,10 @@ const flattenSOCObject = (SOCObject: WebsocAPIResponse): (WebsocSchool | WebsocD
                 for (const section of course.sections) {
                     (section as AASection).color = courseColors[section.sectionCode];
                 }
-                accumulator.push(course as AACourse);
+                const courseWithUpdatedAt = course as AACourse;
+                courseWithUpdatedAt.updatedAt = course.updatedAt;
+
+                accumulator.push(courseWithUpdatedAt);
```

Comment:
This is unnecessary. It's assigning an attribute that's already in an object to a copy of that object, which maintains the same type. FYI, it works if I just revert the change.